lib/event-listeners/food-listener.js

```diff
@@ -6,18 +6,38 @@ import { deleteFood } from '../ajax-requests/delete-food.js'
 
 const $ = require('jquery')
 
-let foodObj = $("#food_form").on("submit", function(event) {
+let name = $("#name").val()
```

Comment:
These values will be set when the page first loads, they will not be updated when your on submit gets fired. Therefore you will have stale values when calling `foodFormHasData` and `foodFormData`. I would change these to

```js
let $foodFormName = $("#food-form-name")
let $foodFormCalories = $("#food-form-calories")
``` 

Then you can refer to these elements inside of those methods calling `.val()` on them there. `.val()` gets the value of the Element at the time it is called.

I would rename the `ids` to be a bit more specific avoid possible overlap of common names like food and calories in the project.

Reply:
ah this is perfect! We ended up realizing this was an issue after pushing it up, but this will help us realize the original intent to dry things up.